.github/workflows/pg-13-migtests.yml

```diff
@@ -70,10 +70,12 @@ jobs:
           cd installer_scripts
           yes | ./install-yb-voyager --install-from-local-source --only-pg-support
           sudo rm /usr/bin/pg_dump
-          sudo ln -s /usr/lib/postgresql/16/bin/pg_dump /usr/bin/pg_dump
+          sudo ln -s /usr/lib/postgresql/17/bin/pg_dump /usr/bin/pg_dump
```

Comment:
I believe you shouldn't change this for 13 pipeline as now with this you will pg_restore 17 on PG13 which will not have the paramter. So if you skip these changes it should fine